Z2A SwiftUI/NSCoffee/NSCoffee/Models/Drinks.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Drink.swift
+//  NSCoffee
+//
+//  Created by Dani on 28/02/2025.
+//
+
```

Comment:
Sorry, this whole file might be a bit messy. I haven't put a lot of thought into it.

Reply:
I was just pleased you hadn't chosen to use CoreData :)